packages/main/test/pages/SideNavigation.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="utf-8">
+
+	<title>SideNavigation</title>
+	<script>
+		delete Document.prototype.adoptedStyleSheets
+	</script>
+
+	<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="../../resources/bundle.esm.js" type="module"></script>
+	<script nomodule src="../../resources/bundle.es5.js"></script>
+
+	<style>
+		html, body {
+			margin: 0;
+			padding: 0;
+		}
+
+		body {
+			display: flex;
+    		flex-direction: column;
+		}
+
+		ui5-side-navigation {
+			height: 100vh;
+			width: 250px;
```

Comment:
Shouldn't the width be according to the longest item or something else automatic?

Reply:
No, there is predefined width, which we are setting on the host and can be modified easily